code-injection-analysis/abiank_redisJSON/web.js

```
@@ -0,0 +1,191 @@
+
+
+// TODO: implement HDEL, DEL
+
+if (process.env.REDISTOGO_URL) {
+    var rtg = require("url")
+        .parse(process.env.REDISTOGO_URL);
+    var redis = require("redis")
+        .createClient(rtg.port, rtg.hostname);
+    redis.auth(rtg.auth.split(":")[1]);
+} else {
+    var redis = require("redis")
```

Biome (2.1.2)

[error] Shouldn't redeclare 'redis'. Consider to delete it or rename it.

'redis' is defined here:


(lint/suspicious/noRedeclare)